content/guardian/Shadowlands-Guardian-FAQ.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Guardian 9.0.2 Frequently Asked Questions
+date: '2020-12-06'
+authors: ['Thyme']
+published: true
+patch: "9.0.2"
+description: "The Most commonly asked questions, Shadowlands Guardian Edition."
+---
+
+Shadowlands has finally arrived! Nartually theere are many questions to be had so here we go.
+# Covenants
+
+**What covenant should I choose?** 
+
+***IT IS IMPORTANT TO STATE THAT EVERY COVENANT WORKS WELL, AND THAT YOU CAN PICK THE ONE YOU PREFER AND NOT GIMP YOURSELF***
+
+* Every covenant is within 5 to 7% of each other depending on the situation (ST/cleave/AOE) (Please refer to the [sims](https://www.dreamgrove.gg/sims/bear/) for more details)
+
+* Night Fae is likely the ideal choice if you are going to Offspec as it is the best for resto and very good for feral, it also offers another burst damage/survivability cooldown, that said in bear form the spell can feel random at times, and you cannot control if you get more damage/surv. You can also use it in other form for more damage or another healing cooldown 
+
+* Necrolord offers one of the best priority damage out of the covenant we have access to, some self healing capabilities if you cast the Swarm on yourself. But will drop off as the target count increases. That is in part mitigated with the soulbind they have access to.
+
+* Kyrian gives you an ability that can give you either strong defensive capabilities/utility or some offensive potential,and, unlike NF you can control which one you get. It can have a few downside (relying on other people, the dps tether acting as a 20% ignore pain and thus can end up wasting a bit of the damage). But unlike necrolord its dps is constant and does not drop off either in ST or AOE. Furthermore Pelagos is still a strong soulbind. 
+Important to note that if you can avoid it you should never use the solo bond of the ability as it is inferior to any other benefits.
+
+* Venthyr gives you Ravenous frenzy, which if paired with incarn probably gives you one of the strongest 3 minute burst in the game, has virtually no downside (you have the gcd+2sec before getting stunned while being idle and bears are gcd locked).Venthyr also has access to strong soulbind for ST, where it will struggle a bit more than other covenant.
+
+TLDR; Pick the one you prefer it’s very close and is unlikely to make or break what you are doing 
+
+### Nightfae
+
+**ALERT** Dreamweaver/Podtender is a trap for tanks, it may or may not be a bug but for the time being do not take it!
+
+When the cheat death is activated, all threat is reset thus redirecting every mob you previously had to your group. Whilst in the pod you are unable to be battle resurrected making you watch your group suffer.
+
+*This may be less relevant in raids, but is a big red flag for m+*
+
+### How To Kyrian Tether
+
+
+First off, your base spell {{< spell 326434 "Kindred Spirits" >}}) (2,5s cast time, only usable in caster form)
```

Comment:
A lot of the spells in this section have an extra `)` after the spell link, seems like this was unintended.